app/dmps/[...slug]/dmps.scss

```diff
@@ -1,464 +1,464 @@
 
```

Comment:
Why not just delete this file? It looks like everything is now commented out.

Reply:
@briri it was deleted in the other TypeAhead branch already, so I didn't bother doing it here